src/bartiq/integrations/__init__.py

```diff
@@ -14,12 +14,9 @@
 
 try:
     from .jupyter.routine_explorer import explore_routine
-
-    additional_all = ["explore_routine"]
 except ImportError:
-    additional_all = []
+    pass
 
 from .latex import routine_to_latex
-from .qref import bartiq_to_qref, qref_to_bartiq
 
-__all__ = ["bartiq_to_qref", "qref_to_bartiq", "rotine_to_latex"] + additional_all
+__all__ = ["routine_to_latex", "explore_routine"]
```

Comment:
Why this change?

Reply:
So:
- There are no `bartiq_to_qref` and `qref_to_bartiq` functions anymore.
- There was a typo in "rotine_to_latex"
- Expression of the form `__all__ = a + b` is an incorrect expression for `__all__` variable (see [here](https://microsoft.github.io/pyright/#/typed-libraries?id=library-interface))

Having said that, we can do better, see [66cc331](https://github.com/PsiQ/bartiq/pull/115/commits/66cc3318ad23a7a97201df42489bcd493931f883)